spinalcordtoolbox/deepseg/core.py

```diff
@@ -22,71 +22,12 @@
 # Default values if not asked during CLI call and if not present in json metadata.
 DEFAULTS = {
     'thr': 0.9,
-    'largest': 1,
-    'fill_holes': 1,
+    'largest': 0,
+    'fill_holes': 0,
+    'remove_small': '0vox'
```

Comment:
I was wondering if we should resort to  `None` instead of 0 as default values. Both ways work with this [sister PR](https://github.com/ivadomed/ivadomed/pull/486/files).

Reply:
None triggers an error during parsing: `TypeError: argument of type 'NoneType' is not iterable`, so I left '0vox' since it gives the same result :)

I also included 'fname_prior' in options here: 9c936c3 (change made in sister PR)